frontend/lib/src/components/widgets/DataFrame/DataFrame.tsx

```diff
@@ -552,12 +546,12 @@ function DataFrame({
         contentAlign: "center",
         allowOverlay: false,
         themeOverride: {
-          textDark: theme.textLight,
+          textDark: gridTheme.glideTheme.textLight,
```

Comment:
nit: it might make sense to destructure `gridTheme` once because there is a lot of access of it:

```javascript
const { glideTheme, headerIcons, maxColumnWidth, maxColumnAutoWidth, minColumnWidth, tableBorderWidth } = gridTheme
```

Reply:
hmm, the full `gridTheme` object is needed in the `useTableSizer` hook, so I would likely need the full object anyways. Also, if you only need a small subset of props from an object, it's great to use destructure. But in this case, it feels to me that it would make the code slightly more complicated to understand. If accessed via `gridTheme.` it is very explicit where the properties belong to.

Reply:
I extracted the `glideTheme` since it is the prop thats used multiple times, but left the others to be accessed from `gridTheme`.

Reply:
I think I like that variant the least. Destructuring indicates to me as a developer that the variable will not be used otherwise anymore, so it will imho set the wrong expectation when reading the code

Reply:
Ok, I reverted it 👍